docs/config/deployer.md

```diff
@@ -12,14 +12,23 @@ AWS profile to use for the deployment. Also used for all AWS CLI commands run lo
 
 AWS region to use for the deployment.  See the [AWS docs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html) for more information.
 
+## AWSAvailabilityZone
+
+AWS Availability Zone in which to deploy instances. See the [AWS docs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html) for more information.
+
+> [!TIP]
+> Deploying instances in the same Availability Zone can greatly reduce traffic costs. Refer to [data transfer pricing](https://aws.amazon.com/ec2/pricing/on-demand/#Data_Transfer_within_the_same_AWS_Region) for more details.
+
+> [!NOTE]
+> RDS clusters generally have a multi-az requirement which means it's not possible to force all instances (e.g. writer, readers) to be on the same availability zone.
```

Comment:
Should we remove this given the new insight?

Reply:
Yeah, I think we can remove it for now. I believe pure RDS clusters still impose this requirement, but we use Aurora so it shouldn't be an issue worth raising.